decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -179,6 +179,7 @@ def profiling_gems
       end
 
       def tweak_spring
+        run "bundle exec spring stop"
```

Comment:
This is necessary because if not you'll have a non-existing directory error in some cases. I could reproduce it in `develop` too, but I couldn't do it in `release/0.28-stable`, so we don't need to backport this fix.